src/ui/components/address-action/AddressActionDetails/AddressActionDetails.tsx

```diff
@@ -36,10 +38,11 @@ export function AddressActionDetails({
         <RecipientLine
           recipientAddress={recipientAddress}
           chain={chain}
+          showNetworkIcon={!showApplicationLine}
```

Comment:
We want to display network icon in the recipient line only if we are not displaying the application line